presidio-anonymizer/presidio_anonymizer/entities/analyzer_result.py

```diff
@@ -0,0 +1,101 @@
+"""
+AnalyzerResult is the exact copy of the recognizer result.
+
+Represents the findings of detected entity.
+"""
+from presidio_anonymizer.entities.invalid_exception import InvalidParamException
+
+
+class AnalyzerResult(object):
+    """
+    AnalyzerResult is a duplication of the recognizer result.
```

Comment:
This line is not required

Reply:
Sadly, the linter thinks otherwise.

Reply:
Let me rephrase: The text in this docstring is unclear.
```suggestion
    AnalyzerResult is the output of the analyze process.
```